docs/ldml/tr35.md

```diff
@@ -3170,6 +3173,41 @@ Processes sometimes encounter invalid number or date patterns, such as a number
 * For a pattern that contains a currently-invalid pattern character (applies only to date patterns, for which A-Za-z are reserved as pattern characters but not all defined as valid):
   * Produce an error (set an error code or throw an exception) when an attempt is made to create a formatter with such a pattern or to apply such a pattern to an existing formatter.
 
+## 8 <a name="Data_Size" href="#Data_Size">Data Size Reduction</a>
+Software implementations may have constrained memory requirements. The following outlines some techniques for filtering out CLDR data for a particular implementation. The exact filtering would depend on the particular requirements of the implementation in question, of course.
+
+Locale data can be _sliced_ to exclude data not needed by a particular implementation. This can be _vertical slicing_: excluding a locale and all the locales inheriting from them, or _horizontal slicing_: excluding particular types of data from all locales. For example:
+  * A vertical slice could retain only those locales used in a particular set of markets, such as EU locales.
+  * A horizontal slice could remove all data in the emoji/ directory, which are annotations for emoji and symbols.
+  
+Of course, both of these techniques can be applied.
+
+### 8.1 <a name="Vertical_Slicing" href="#Vertical_Slicing">Vertical Slicing</a>
+
+It is important to note that if a particular locale is in a vertical slice, then all of its parents should be as well, because of inheritance. This is not a factor if the data is fully resolved, as in Javascript.
```

Comment:
```suggestion
It is important to note that if a particular locale is in a vertical slice, then all of its parents should be as well, because of inheritance. This is not a factor if the data is fully resolved, as in the JSON format data.
```

correct name